client/fingerprint/cpu.go

```diff
@@ -42,7 +42,7 @@ func NewCPUFingerprint(logger hclog.Logger) Fingerprint {
 }
 
 func (f *CPUFingerprint) Fingerprint(request *FingerprintRequest, response *FingerprintResponse) error {
```

Comment:
There's a test failure covering this method:

```
=== FAIL: client/fingerprint TestCPUFingerprint_OverrideCompute (0.00s)
    cpu_default_test.go:108: expected override cpu of 4711 but found 4588
```

And a test compilation error:

```
Error: client/stats/cpu_test.go:35:13: not enough arguments in call to shelpers.Init
	have ()
	want (uint64)
```